util/Migrator/DbScripts/2024-07-01_00_EnsureMaxGBAndSeatsAreSet.sql

```diff
@@ -0,0 +1,8 @@
+UPDATE
```

Comment:
💭 Just double checking: we don't ever **allow** `null` values for those properties, right? So we can make the assumption (as your migration is doing here) that they should always be default to `1`.

Reply:
I did some digging with @rkac-bw to make sure that these were valid. Seats being null doesn't really happen in production, so I dropped setting it in the last commit. MaxStorageGb should be defaulted to 1 for all _premium_ organizations instead of all plan types as I originally had here, so made that update as well